jekyll/_cci2/customizing-ios.md

```diff
@@ -0,0 +1,429 @@
+---
+layout: classic-docs
+title: Testing iOS Applications on macOS
+short-title: Testing iOS Applications on macOS
+categories: [configuring-jobs]
+description: Testing iOS applications on macOS
+order: 71
+---
+
+This document describes how to set up and customize testing for an iOS application with CircleCI in the following sections:
+
+* TOC
+{:toc}
+
+## Overview
+
+CircleCI offers support for building and testing iOS and macOS projects. Refer to the manifest of the software installed on CircleCI OSX and macOS build images in the Using a macOS Build Image document.
+
+## Getting Started
+
+Select a macOS project you would like to build on the [Add
+Projects page](https://circleci.com/add-projects). **Note:** If your project does not appear on the OS X tab, it may be listed as a Linux project. Add your project as a Linux build, then change
+it to a macOS build by enabling the Build OS X project option on the
+Project Settings > Build Environment page.
+
+## Basic setup
+
+After enabling macOS builds for your project, share
+the scheme that is going to be built on CircleCI so that we run the
+correct build actions. Here is how to share an existing scheme in Xcode:
+
+1. Choose Product > Scheme > Manage Schemes.
+2. Select the Shared option for the scheme to share, and click Close.
+3. Choose Source Control > Commit.
+4. Select the Shared Data folder.
+5. Enter your commit message in the text field.
+6. Select the "Push to remote" option (if your project is managed with Git).
+7. Click the Commit Files button.
+A new `.xcscheme` file is located in the
+`xcshareddata/xcschemes` folder under your Xcode project. 
+8. Commit this file to your git repository so that CircleCI can access it.
+
+Simple projects should run with minimal or no configuration.
+
+By default, CircleCI will:
```

Comment:
As there is no inference, none of these items apply to 2.0, we should delete them